README.md

```diff
@@ -88,6 +88,17 @@ Journaling provides a number of different configuation options that can be set i
   This can be used to configure what `priority` the Delayed Jobs are enqueued with. This will be applied to all the Journaled::Devivery jobs that are created by this application.
   Ex: `Journaled.job_priority = 14`
 
+#### DJ `enqueue` options
+
+Both model-level directives accept additional options to be passed into DelayedJob's `enqueue` method:
+
+```ruby
+# For change journaling:
+journal_changes_to :email, as: :identity_change, priority: 10
+
+# Or for custom journaling:
+journal_attributes :email, priority: 20, queue: 'journaled'
```

Comment:
should we prefix these flags with `job_` or something (e.g. `job_priority`, `job_queue`)? When we're in the event delivery business, `queue` might be confused with, e.g., the kinesis stream we're publishing to?

Reply:
How about `delayed_job: { priority: 20, queue: 'journaled' }`?

Reply:
`enqueue_with: { priority: 20, queue: 'journaled' }`?

Reply:
i like a single kwarg for `journal_attributes` or `journal_changes_to` that contains all delayed_job options, seems like it's very clear what they are for and it simplifies the contract a bit

Reply:
agreed. To be specific about your two examples, I like `enqueue_with` I think because it feels like we'd ultimately want to be providing a job queue-agnostic adapter, and it seems like we should validate that there exists exactly one job queue arg-bag that we can strongly validate against the capabilities of the current adapter so nobody has phantom configs that aren't being read for a job adapter they're not using.